src/views/download-banner/DownloadBanner.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+import Button from "../../components/Button";
+import ConnectWallet from "../connect-wallet/ConnectWallet";
+import AppNav from "../app-nav/AppNav";
+
+import { useGlobalState } from "../../AppStateHolder";
+import useReferral from "../../hooks/useReferral";
+import { refereeTiers } from "../../hooks/useRewards";
+
+export default function DownloadBanner(props: unknown) {
+  const { canAccessApp } = useGlobalState();
+
+  const [shape, setShape] = useState("rect");
+  const [url, setUrl] = useState("");
+
+  const urlPrefix = "https://perp.ml/refBanner";
+
+  const { referralCode, vipTier } = useReferral();
+
+  useEffect(() => {
+    const discount = refereeTiers[vipTier]["rebate"] * 100;
```

Comment:
Need clarity here to figure out what rebate does a person who uses this code will get

Reply:
Seems like this part of code is correct and the image should always show "20%" discount at current tier defination